### PR TITLE
ACP-3507 Fixed class name in propel schema.

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Ticket: https://spryker.atlassian.net/browse/ACP-3507

- Release Group: https://release.spryker.com/release-groups/view/5558

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   AppMerchant           | patch                 |                       |

-----------------------------------------

##### Change log

Fixes

- Updated PHP Version.
- Fixed schema files.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,35 +14,35 @@
...
+        "spryker/event-dispatcher": "^1.6.0",
         "spryker/glue-backend-api-application": "*",
-        "spryker/glue-json-api-convention": "^1.3",
-        "spryker/http": "^1.11",
+        "spryker/glue-json-api-convention": "^1.3.0",
+        "spryker/http": "^1.11.0",
         "spryker/message-broker-aws": "*",
         "spryker/propel": "*",
-        "spryker/router": "^1.19",
+        "spryker/router": "^1.19.0",
```
Comment (matweew) [inline composer.json]: potential conflict with https://github.com/spryker/app-merchant/pull/9/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34

File: src/Spryker/Zed/AppMerchant/Business/Message/MessageSender.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ public function sendMerchantAppOnboardingStatusChangedMessage(
         MerchantTransfer $merchantTransfer,
         AppConfigTransfer $appConfigTransfer
     ): void {
-        $merchantOnboardingStatus = $merchantTransfer->getConfigOrFail()[AppMerchantConfig::MERCHANT_ONBOARDING_STATUS];
+        $merchantOnboardingStatus = $merchantTransfer->getConfig()[AppMerchantConfig::MERCHANT_ONBOARDING_STATUS];
```
Comment (matweew) [inline src/Spryker/Zed/AppMerchant/Business/Message/MessageSender.php:39]: no default value?

Comment (stereomon) [reply src/Spryker/Zed/AppMerchant/Business/Message/MessageSender.php:39]: Wasn't there before. Only CI failures were fixed by this.

File: composer.json (hunk 1 above)
Comment (stereomon) [reply composer.json]: Any action from my end you see here?

Comment (matweew) [reply composer.json]: synchronize packages list in require and required-dev

File: composer.json [hunk 3]
```diff
@@ -4,45 +4,45 @@
...
         "spryker/glue-application-extension": "^1.0.0",
+        "spryker/glue-backend-api-application": "*",
+        "spryker/glue-json-api-convention": "^1.3.0",
+        "spryker/http": "^1.11.0",
         "spryker/kernel": "^3.30.0",
         "spryker/log": "^3.0.0",
         "spryker/message-broker": "^1.11.0",
-        "spryker/transfer": "^3.33.0"
+        "spryker/message-broker-aws": "*",
+        "spryker/propel": "*",
```
Comment (matweew) [inline composer.json]: `*` usually have to be changed to a minimal possible version.

**Review by matweew — APPROVED** [review]

File: composer.json (hunk 3 above)
Comment (stereomon) [reply composer.json]: Missed it, updated.

**Pull request merged**